README.md

```diff
@@ -20,7 +20,7 @@ Note: This is not an officially supported Google product
 
 ## Building
 
-Marl contains a number of unit tests and examples which can be built using CMake.
+Marl contains many unit tests and examples that can be built using CMake.
```

Comment:
There are plenty of unit tests, but aren't many examples (_currently_). While I'd like to improve this, I think this may be misleading.

`which` -> `that` is good, thanks.